mod/developers/views/default/theme_sandbox/modules/widgets.php

```diff
@@ -52,17 +52,19 @@ function css_permissions_override() {
 
 ?>
 </div>
-<script type="text/javascript">
+<script>
+require(['elgg', 'jquery'], function (elgg, $) {
 	// widgets do not have guids so we override the edit toggle and delete button
 	$(function() {
 		$('.elgg-widget-edit-button').unbind('click');
-		$('.elgg-widget-edit-button').click(function() {
+		$('.elgg-widget-edit-button').on('click', function() {
```

Comment:
I think we should really start delegating events through $(document) to avoid all the boilerplate with binding and unbinding. Then we can just check if something has been instantiated, and if so terminate early. It's a lot friendlier with AJAX.